### DISCUSSION TITLE
Sort map (deck.gl Scatterplot) legend in Apache Superset

### DISCUSSION THREAD

**Discussion by mona-mk** [apache/superset#18452] [Ideas]
Map legend needs to be sorted alphabetically or even better with a custom given order (same applies with numerical legends). e.g. in example image below, I want the order A, B, C, D for the legend.

<img width="636" alt="Screenshot 2021-02-24 at 13 47 15" src="https://user-images.githubusercontent.com/59080347/108996367-f4e18080-76a6-11eb-814f-6033ca12b75c.png">

**Comment by santosh-sahoo334** [comment]
Hi, Is there any workaround on this ? I am looking for similar setup. Superset v2.0.0

**Comment by tabgruedksr** [comment]
Has there been any progress or workaround for this? Still facing the same issue.
